rhino/rhino-test.config

```diff
@@ -0,0 +1,12 @@
+# This file can be used to enable certain options in this module for testcases.
+#
+# for example, if there is an (experimental) feature xyz, you can set
+# rhino.xyz.enabled=true
+# and write tests for features, whiche are not enabled by default for production builds
+#
+#
+# Please keep this values for RhinoPropertiesTest
+test.file.rhino-test-config.loaded=true
+test.config.bar=value4-mod
```

Comment:
This file is a placeholder/for documentation and is also used to demonstrate, that a top-level file can override values in a classpath file.
Should I delete it (and the related test)?